utils/findutils/Makefile

```diff
@@ -32,56 +35,54 @@ sufficient functionality, but some users may want or need
 the full functionality of the GNU tools.
 endef
 
-define Package/findutils-find
-	$(call Package/findutils/Default)
-	TITLE+= (find)
-endef
-
-define Package/findutils-find/description
-$(call Package/findutils/description/Default)
-This package contains the find utility
+define Package/findutils
+	TITLE+= findutils (all)
+	DEPENDS:= \
+	+findutils-find \
+	+findutils-xargs \
+	+findutils-locate
 endef
 
-define Package/findutils-xargs
+define Package/findutils-find
 	$(call Package/findutils/Default)
-	TITLE := (xargs)
-endef
-
-define Package/findutils-xargs/description
-$(call Package/findutils/description/Default)
-This package contains the xargs utility
+	TITLE+= find utility
 endef
 
 define Package/findutils-locate
 	$(call Package/findutils/Default)
-	TITLE := (locate)
+	TITLE := locate and related updatedb utility
```

Comment:
Should't this be +=  just like "find utility" ?

Reply:
You're correct! I haven't noticed it during re-writing as it was there. However, I need to remove word "related" otherwise the title was too long, so it disappeared from `make menuconfig`.